### DISCUSSION TITLE
Is there any way to change the font of Normal text?

### DISCUSSION THREAD

**Discussion by FrozenSky7124** [excalidraw/excalidraw#7061] [Q&A] [answered]
I have changed the font of Hand-draw and Code text by modify woff2 files in "public" dir.
But, I can't find how to change font of  Normal text. Is there any way to do that?

![exdrawfont](https://github.com/excalidraw/excalidraw/assets/4454570/5285c3f8-eb6d-4717-8e04-1eb3d1fa675b)

**Comment by FrozenSky7124** [comment, marked as answer]
I have done this. 

Add codes to font.css
```
@font-face {
  font-family: "Helvetica";
  src: url("font_LXGW.woff2");
  font-display: swap;
}
```

Add codes to index.html
```
    <link
      rel="preload"
      href="/font_LXGW.woff2"
      as="font"
      type="font/woff2"
      crossorigin="anonymous"
    />
```

**Comment by ashishrai888** [comment]
is there any way to change font-style in excalidraw . I am using cdn link of excalidraw . is there any way to override css for font-style change . By default it provide 3 font-style

**Reply by DawidWraga** [reply]
I spent a while figuring this out but I've found a solution.

I've written an article on how to use custom fonts on excalidraw [here](https://dev.to/dawidcodes/how-to-use-a-custom-font-on-excalidrawcom-4jl4)

**Reply by LDSmj** [reply]
I fixed mine by adding currentItemFontFamily: 2 to my appState. So it will look like this:

  appState: {
      currentItemFontFamily: 2
  }
